pkg/proc/target_exec.go

```diff
@@ -0,0 +1,888 @@
+package proc
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/token"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/go-delve/delve/pkg/dwarf/reader"
+)
+
+// Next continues execution until the next source line.
+func (t *Target) Next() (err error) {
```

Comment:
I'd like to keep the name of the recevier as dbp here and in the other functions to preserve git history.

Reply:
Also put (*Target).Next right before (*Target).Step so they are ordered logically.